tests/test_minkowski_space_tensorflow.py

```diff
@@ -0,0 +1,301 @@
+"""
+Unit tests for Minkowski space.
+"""
+
+import importlib
+import math
+import numpy as np
+import os
+import tensorflow as tf
+import sys
+sys.path.append("~/Desktop/")
+import geomstats.backend as gs
+import tests.helper as helper
+
+from geomstats.minkowski_space import MinkowskiSpace
+
+
+class TestMinkowskiSpaceTensorFlow(tf.test.TestCase):
+    _multiprocess_can_split_ = True
+
+    def setUp(self):
+        gs.random.seed(1234)
+
+        self.time_like_dim = 0
+        self.dimension = 2
+        self.space = MinkowskiSpace(self.dimension)
+        self.metric = self.space.metric
+        self.n_samples = 10
+        
+    @classmethod
+    def setUpClass(cls):
+        os.environ['GEOMSTATS_BACKEND'] = 'tensorflow'
+        importlib.reload(gs)
+
+    @classmethod
+    def tearDownClass(cls):
+        os.environ['GEOMSTATS_BACKEND'] = 'numpy'
+        importlib.reload(gs)
+
+    def test_belongs(self):
+        point = self.space.random_uniform()
+        belongs = self.space.belongs(point)
+        expected = tf.convert_to_tensor([[True]])
+
+        with self.test_session():
+            self.assertAllClose(gs.eval(belongs), gs.eval(expected))
+    
+    def test_random_uniform(self):
+         point = self.space.random_uniform()
+         with self.test_session():
+             self.assertAllClose(gs.eval(point).shape, (1, self.dimension))
+
+    def test_random_uniform_and_belongs(self):
+        point = self.space.random_uniform()
+        with self.test_session():
+            self.assertTrue(gs.eval(self.space.belongs(point)))
+
+    def test_inner_product_matrix(self):
+        result = self.metric.inner_product_matrix()
+ 
+        expected = tf.convert_to_tensor([[-1.0, 0.], [0., 1.]])
+        with self.test_session():
+            self.assertAllClose(gs.eval(result), gs.eval(expected))
+  
+    def test_inner_product(self):
+        point_a = tf.convert_to_tensor([0., 1.])
+        point_b = tf.convert_to_tensor([2., 10.])
+ 
+        result = self.metric.inner_product(point_a, point_b)
+        expected = helper.to_scalar(gs.dot(point_a, point_b))
+        
+        with self.test_session():
+            expected -= (2 * point_a[self.time_like_dim]
```

Comment:
Do we need this line to be inside the session? If not, put it above so that the syntax is consistent with the other tests.